notebooks/en/fine_tuning_vit_custom_dataset.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>Is this dataset available on the Hub? I think it'd be easier for users to follow along if they could also download the dataset or if you provided some more information/details about how a user can create their own dataset with their images.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Dataset is not in Hub, it's on my computer. However people can find lots of similar datasets from Kaggle